### ISSUE TITLE
heif / hevc support

### ISSUE THREAD

**Issue by fredgolightly** [kollerlukas/Camera-Roll-Android-App#148]
Android P will support this, and I think a fair few of us will switch over for better quality/smaller sizes, so support in advance would be awesome!

**Comment by yui-haider** [comment]
Version 1.0.6 actually does somehow support HEIC. (At least it works on my machine)

I created a HEIC image with GIMP and transfered it to my phone. It didn't show up.
But when I changed the file extension to .jpg and opened it with camera roll it worked. And it also does show up in the gallery with preview. Only additional file information didn't work. (Didn't work means it doesnt show correct information but doesn't crash)

So just adding .heic to the supported file format list and it should be possible to view HEIC images.

**Comment by fredgolightly** [comment]
> 
> 
> Version 1.0.6 actually does somehow support HEIC. (At least it works on my machine)
> 
> I created a HEIC image with GIMP and transfered it to my phone. It didn't show up.
> But when I changed the file extension to .jpg and opened it with camera roll it worked. And it also does show up in the gallery with preview. Only additional file information didn't work. (Didn't work means it doesnt show correct information but doesn't crash)
> 
> So just adding .heic to the supported file format list and it should be possible to view HEIC images.

Can you zoom 100%? I suspect this may just be reading an embedded thumbnail.

**Comment by yui-haider** [comment]
Yes I can zoom to 100%. I can even use the function "use as" and use the image as wallpaper.

I tried to implement this feature, but I struggle with setting up the project in Android Studio.
